src/main.rs

```diff
@@ -69,6 +68,8 @@ struct UploadArgs {
     team: Option<String>,
     #[arg(long, help = "Value to override CODEOWNERS file or directory path.")]
     codeowners_path: Option<String>,
+    #[arg(long, help = "Run commands without the quarantining step.")]
```

Comment:
If default is `false`, then `--enable-quarantining` should say `Run commands with the quarantining step.`